python/glow/gwas/log_reg.py

```diff
@@ -72,19 +74,27 @@ def logistic_regression(
         - ``tvalue``: The chi squared test statistic according to the score test or the correction method
         - ``pvalue``: P value estimated from the test statistic
         - ``phenotype``: The phenotype name as determiend by the column names of ``phenotype_df``
+        - ``effect``: The effect size (if approximate Firth correction was applied)
```

Comment:
nit: these should be in the same order as they appear in the output struct. Below it looks like we're putting these columns first.